README.rst

```diff
@@ -116,6 +119,52 @@ Usage
       latest     Get the latest tag in a changelog.
       tag        Create an annotated git tag based on a changelog entry.
       update     Updates the version and date of the most recent changelog entry.
+      create     Creates a new CHANGELOG.md file with a standard header.
+      entry      Add a quick entry to the current version.
+      next       Create a new unreleased version in changelog file.
+
+next
+----
+
+Create a new unreleased version in changelog file.
+
+::
+
+   Usage: chag next
+
+   Creates a new unreleased version inside of the changelog file.
+
+   Options:
+     --help     Displays this message.
+
+entry
```

Comment:
I am not suggesting that my changes should not be used by other people, what I am saying is that including my changes in your own PR and then claiming this one is in so many words "deprecated" is not polite because you never asked me if that was the case.

Considering this PR is still active and I'm actively making changes to it as they are requested by repo owner I think that it would have been ideal if you had either tried to collaborate to modify this PR or at least discussed your suggestions prior to just commenting on this with "Superseded by...".

I appreciate that you think these changes are good enough to want to use them (after all that's the spirit of open source!) but we should be polite in our approach to collaborating and not aggressive, so not to be misunderstood.

Besides the points I am making above, it seems that your PR is adding other unrelated functionality such as a config file, that really doesn't have much to do with what I am doing here but could be useful in it's own right.